creme/neighbors/__init__.py

```diff
@@ -1,9 +1,13 @@
 """Neighbors-based learning."""
 from .knn import KNeighborsClassifier
 from .knn import KNeighborsRegressor
```

Comment:
I guess you could remove these classes in your PR :)

Reply:
I'll do that as we progress with the PR. Currently I am using them for comparison and reference :D

I created a notebook comparing the `dict`-based implementation against the numpy+ckDTree one. So far we are getting a speed up of about 9x, depending on the window size!